analyzer/android/lib/api/droidbot/common.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (C) 2012-2015  Diego Torres Milano
```

Comment:
Just to be clear, it is necessary for `DroidBot` to run *in* the emulator, yes? We can't run it from the outside through `adb` or so? And then, would it be possible to install a `Droidbot` Python package during the device/emulator setup, rather than embedding it directly in Cuckoo?